Simplenote.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,25 @@
+{
+  "object": {
+    "pins": [
+      {
+        "package": "SimplenoteFoundation",
+        "repositoryURL": "git@github.com:Automattic/SimplenoteFoundation-Swift.git",
+        "state": {
+          "branch": null,
+          "revision": "c5938f97ba6c041ae4c0f4030493f628f2263a41",
+          "version": "1.0.0"
+        }
+      },
+      {
+        "package": "SimplenoteSearch",
+        "repositoryURL": "git@github.com:Automattic/SimplenoteSearch-Swift.git",
+        "state": {
+          "branch": null,
+          "revision": "c594338829e8fb8954fad8743508b135da17b132",
+          "version": "1.0.0"
+        }
+      }
+    ]
+  },
+  "version": 1
+}
```

Comment:
It's a bit odd for few reasons!!

- Not present in `develop`
- SimplenoteFoundation is pinned to 1.1.0 actually
- And there's a third dependency called SimplenoteInterlinks

Should we just delete this file? (maybe it's a stale thing generated, at some point, by Xcode 11?)

Reply:
@jleandroperez interesting... I thought it was an Xcode 12 thing but I clearly got confused with [the `Package.resolved` from `Simplenote.xcworkspace`](https://github.com/Automattic/simplenote-macos/blob/51dfca83a4d1d1d8c0e7ceee6d9b0e069f0a91e3/Simplenote.xcworkspace/xcshareddata/swiftpm/Package.resolved) which we already track. Having two wouldn't make much sense...

I deleted the file and reopened the workspace, waiting for the Swift packages to resolve. The file was not created. Seems safe to remove – which is what I did with an interactive rebase.

Thanks!